x-pack/legacy/plugins/lens/public/xy_visualization/xy_suggestions.ts

```diff
@@ -373,7 +373,9 @@ function buildSuggestion({
 
   return {
     title,
-    score: getScore(yValues, splitBy, changeType),
+    score: currentState
+      ? getScore(yValues, splitBy, changeType)
```

Comment:
nitpick: we could move the function execution up to not repeat the code? I'd also multiply by 0.5 instead of dividing as the division is heavy on performance. That's a micro optimization so feel free to ignore both.

```
const score = getScore(yValues, splitBy, changeType)
 return {
    title,
    score: currentState 
		? score 
		: score * 0.5 // Lower score for non-XY charts
```